### ISSUE TITLE
[Old] Spell Check failed after adding GWL

### ISSUE THREAD

**Issue by charliehoward4dp** [DistributedProofreaders/guiguts#1200]
Started the old Spell Checker, clicked "add good words"; when nothing seemed to happen, I pressed F7 again, and eventually the Message Log appeared with these messages:

```
15:21:17: Guiguts 1.6-test2: If you have any problems, please include any error messages that appear here with your bug report.
16:35:03: Tk::Error: not a Tk object
 Tk::die_with_trace at D:/Documents/DistrProof/PP/GG 1.6.0-test2/lib/Guiguts/SpellCheck.pm line 350
 Guiguts::SpellCheck::spellshow_guesses at D:/Documents/DistrProof/PP/GG 1.6.0-test2/lib/Guiguts/SpellCheck.pm line 120
 Guiguts::SpellCheck::spellchecknext at D:/Documents/DistrProof/PP/GG 1.6.0-test2/lib/Guiguts/SpellCheck.pm line 42
 Guiguts::SpellCheck::spellcheckfirst at D:/Documents/DistrProof/PP/GG 1.6.0-test2/lib/Guiguts/SpellCheck.pm line 517
 Guiguts::SpellCheck::spellchecker at D:/Documents/DistrProof/PP/GG 1.6.0-test2/lib/Guiguts/KeyBindings.pm line 52
 Guiguts::KeyBindings::__ANON__ at C:/Strawberry/perl/site/lib/Tk.pm line 424
 (eval) at C:/Strawberry/perl/site/lib/Tk.pm line 424
 Tk::MainLoop at D:\Documents\DistrProof\PP\GG 1.6.0-test2\guiguts.pl line 387
 Tk::die_with_trace at C:/Strawberry/perl/site/lib/Tk.pm line 488
 Guiguts::SpellCheck::spellshow_guesses at D:/Documents/DistrProof/PP/GG 1.6.0-test2/lib/Guiguts/SpellCheck.pm line 350
 Guiguts::SpellCheck::spellchecknext at D:/Documents/DistrProof/PP/GG 1.6.0-test2/lib/Guiguts/SpellCheck.pm line 120
 Guiguts::SpellCheck::spellcheckfirst at D:/Documents/DistrProof/PP/GG 1.6.0-test2/lib/Guiguts/SpellCheck.pm line 42
 Guiguts::SpellCheck::spellchecker at D:/Documents/DistrProof/PP/GG 1.6.0-test2/lib/Guiguts/SpellCheck.pm line 517
 Guiguts::KeyBindings::__ANON__ at D:/Documents/DistrProof/PP/GG 1.6.0-test2/lib/Guiguts/KeyBindings.pm line 52
 <Key-F7>
 (command bound to event)
```

**Comment by charliehoward4dp** [comment]
Tried using Spell Query: after clicking "add good words.txt", both the SQ and GG windows froze for about 20 seconds, then the SQ window blanked for a second or two and then displayed the list again. The Message Log didn't appear.

Perhaps the errors shown with Spell Check were caused by my pressing F7 while the Tool was frozen (or processing for a really long time).  Yes; that's probably the explanation: I did the same thing with SQ, pressing Ctrl-F7 while the windows were frozen, and when they became responsive again, the Message Log appeared.

Since I don't think the project text file matters, here are just the GWL and BWL, in case something in one of them is causing this.

[gwl.zip](https://github.com/DistributedProofreaders/guiguts/files/11521110/gwl.zip)
